ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/terminattr-prem-no-ingestkey.yml

```diff
@@ -1,13 +0,0 @@
-### Daemon TerminAttr
-daemon_terminattr:
-  cvaddrs:
-    - 10.10.10.8:9910
-    - 10.10.10.9:9910
-    - 10.10.10.10:9910
-  cvvrf: mgt
-  cvauth:
-    method: "key"
-    # Testing without key variable even set.
```

Comment:
i dont see this case when method is set as key but key variable not set

Reply:
I think it is not really needed. The difference is just as below:
With key set as `magickey`:
```
 exec /usr/bin/TerminAttr -cvaddr=10.10.10.8:9910,10.10.10.9:9910,10.10.10.10:9910 **-cvauth=key,magickey** -cvvrf=mgt -cvgnmi -disableaaa -cvproxy=http://arista:arista@10.10.10.1:3128 -grpcaddr=mgmt/0.0.0.0:6042 -grpcreadonly -smashexcludes=ale,flexCounter,hardware,kni,pulse,strata -ingestexclude=/Sysdb/cell/1/agent,/Sysdb/cell/2/agent -taillogs=/var/log/messages,/var/log/agents/ -cvconfig -cvsourceintf=Vlan100
```
Without empty key:

```
exec /usr/bin/TerminAttr -cvaddr=10.10.10.8:9910,10.10.10.9:9910,10.10.10.10:9910 -**cvauth=key,** -cvvrf=mgt -cvgnmi -disableaaa -cvproxy=http://arista:arista@10.10.10.1:3128 -grpcaddr=mgmt/0.0.0.0:6042 -grpcreadonly -smashexcludes=ale,flexCounter,hardware,kni,pulse,strata -ingestexclude=/Sysdb/cell/1/agent,/Sysdb/cell/2/agent -taillogs=/var/log/messages,/var/log/agents/ -cvconfig -cvsourceintf=Vlan100
```

We required one more key to test this empty key.

Reply:
yeah but without knowing the history we should not move this test from molecule. Even this is going to require to improve the coverage

Reply:
Added.